### DISCUSSION TITLE
what does the  VideoBandwidth=2000 in the slinger config file means ?

### DISCUSSION THREAD

**Discussion by flipper2222222** [GerryDazoo/Slinger#433] [Q&A]
hi all, 

does anyone know if VideoBandwidth=2000 in the slinger config file means 2meg upload from the slinger location?

**Comment by mdehrlich** [comment]
I have run some tests.  The resolution is fixed, by your res setting so the slingbox will vary the "refresh" rate to your player to try to hit the 2mbit data rate.   It varies above and below the target, but seems to average at the target.  So your signal from the STB might be 30 or 60 fps, but the H.264 compressed signal to your player is not the same fps.

**Reply by GerryDazoo** [reply]
Interesting. Thanks for digging into this.

**Reply by mdehrlich** [reply]
It means 2Mbit/s target average datarate from the slingbox to the slinger, which then forwards it to your player.

**Reply by flipper2222222** [reply]
> It means 2Mbit/s target average datarate from the slingbox to the slinger, which then forwards it to your player.

thanks  [mdehrlich] for the clear answer

**Reply by swilsonnc** [reply]
> It means 2Mbit/s target average datarate from the slingbox to the slinger, which then forwards it to your player.

But does it also mean 2Mbit/s to the player or is that controlled elsewhere?

**Comment by sp222222** [comment]
always round your numbers by 64. so in this case 2048 or 1920 or 1984. you will get better results
